src/common/test/aws/sdk/kotlin/crt/http/HttpRequestResponseTest.kt

```diff
@@ -8,57 +8,94 @@ package aws.sdk.kotlin.crt.http
 import aws.sdk.kotlin.crt.runSuspendTest
 import aws.sdk.kotlin.crt.util.Digest
 import aws.sdk.kotlin.crt.util.encodeToHex
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
+import org.mockserver.client.MockServerClient
+import org.mockserver.integration.ClientAndServer
+import org.mockserver.model.HttpRequest.request
+import org.mockserver.model.HttpResponse.response
 import kotlin.test.*
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class HttpRequestResponseTest : HttpClientTest() {
     private val TEST_DOC_LINE =
         "This is a sample to prove that http downloads and uploads work. It doesn't really matter what's in here, we mainly just need to verify the downloads and uploads work."
     private val TEST_DOC_SHA256 = "c7fdb5314b9742467b16bd5ea2f8012190b5e2c44a005f7984f89aab58219534"
 
+    val port = 60108
```

Comment:
**suggestion**: Is there not a way to get a random open port rather than hard coding? Similar to how we do it in smithy-kotlin http test suite?

Reply:
There is a way, by setting the port to 0